packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1238,12 +1238,19 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
     final RenderBox button = context.findRenderObject()! as RenderBox;
     final RenderBox overlay = Navigator.of(context).overlay!.context.findRenderObject()! as RenderBox;
     final PopupMenuPosition popupMenuPosition = widget.position ?? popupMenuTheme.position ?? PopupMenuPosition.over;
-    final Offset offset;
+    late Offset offset;
     switch (popupMenuPosition) {
       case PopupMenuPosition.over:
         offset = widget.offset;
       case PopupMenuPosition.under:
-        offset = Offset(0.0, button.size.height - (widget.padding.vertical / 2)) + widget.offset;
+        offset = Offset(
+          0.0,
+          button.size.height,
+        ) + widget.offset;
```

Comment:
nit: maybe we don't have to break line since it is pretty short:)

Reply:
Sure I did it in [refactor: Don't break the line](https://github.com/flutter/flutter/pull/125534/commits/3d710ec23d2d248bef4260eecf5942ddc82df827)